packages/app-store/exchange2016calendar/api/add.ts

```diff
@@ -38,6 +38,7 @@ async function postHandler(req: NextApiRequest, res: NextApiResponse) {
     teamId: null,
     appId: "exchange2016-calendar",
     invalid: false,
+    delegationCredentialId: null,
```

Comment:
Not a fan of prop drilling. Won't block because of this but we should be looking into context stores and dependency injection